### PR TITLE
Header

### PR DESCRIPTION
**Pull request by andylushman**
Header component with `menu-component`

### CODE REVIEW

File: src/components/header/header-component.vue [hunk 1]
```diff
@@ -0,0 +1,60 @@
+<style lang="scss">
```
Comment (prowe214) [inline src/components/header/header-component.vue:1]: for cleanliness you may want to break styles out into a separate scss file and `@import` it here. 

Same for `menu-component.vue`

File: Dockerfile.web [hunk 2]
```diff
@@ -1,11 +1,11 @@
 FROM node:8.9.0 as node
 
-WORKDIR /work
+  WORKDIR /work
```
Comment (prowe214) [inline Dockerfile.web]: Is this a convention?  The whitespace doesn't seem necessary but I'm not very familiar with Dockerfile conventions.

Comment (andylushman) [reply Dockerfile.web]: Good catch! I thought it was suppose to be indented but after some reading, looks like it shouldn't .

File: src/components/header/header-component.vue (hunk 1 above)
Comment (andylushman) [reply src/components/header/header-component.vue:1]: Thanks! Done 👍

**Review by prowe214 — APPROVED** [review]

**Pull request merged**